CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### 2.4.1 (TBD)
 
+- Feature: A new loglevel subcommand was added that enables changing the loglevel temporarily for the local daemons, 
```

Comment:
Not your fault bc 2.4.1 was released after you put up this PR, but move this and the change below into ### 2.4.2 (TBD)

Reply:
I would say also add here or add another `change` block that says the default log level is `info` for all components of Telepresence.